docs/installation/production-rack/aws.md

```diff
@@ -37,10 +37,11 @@ The following environment variables are required:
 
 ### Available Parameters
 
-| Name        | Default       | Description                                      |
-| ----------- | ------------- | ------------------------------------------------ |
-| `cidr`      | `10.1.0.0/16` | CIDR range for VPC                               |
-| `node_disk` | `20`          | Node disk size in GB                             |
-| `node_type` | `t3.small`    | Node instance type                               |
-| `private`   | `true`        | Put nodes in private subnets behind NAT gateways |
-| `region`    | `us-east-1`   | AWS Region                                       |
\ No newline at end of file
+| Name        | Default       | Description                                                           |
+| ----------- | ------------- | --------------------------------------------------------------------- |
+| `cidr`      | `10.1.0.0/16` | CIDR range for VPC                                                    |
+| `node_disk` | `20`          | Node disk size in GB                                                  |
+| `node_type` | `t3.small`    | Node instance type                                                    |
+| `private`   | `true`        | Put nodes in private subnets behind NAT gateways                      |
+| `region`    | `us-east-1`   | AWS Region                                                            |
+| `syslog`    |               | Forward logs to a syslog endpoint (e.g. `tcp+tls://example.org:1234`) |
```

Comment:
```suggestion
| `syslog`    |               | Forward all logs to one or more comma delimited syslog endpoints (e.g. `tcp+tls://example.org:1234` or `tcp+tls://example.org:1234,tcp://example.com:5678) |
```

Reply:
While this functionality is in there trying to keep it simple for now. Not sure how many people would even use this

Reply:
Fair enough, I guess it would be unusual to have multiple logging providers 😆